extensions/windows-event-log/wel/JSONUtils.cpp

```diff
@@ -67,6 +59,29 @@ rapidjson::Value xmlDocumentToJSON(const pugi::xml_node& node, rapidjson::Docume
   return children;
 }
 
+void simplifiedGenericXmlToJson(const pugi::xml_node& node, rapidjson::Value& val, rapidjson::Document& doc, bool flatten = false) {
```

Comment:
these parameters could have better names, e.g `const pugi::xml_node& source, rapidjson::Value& target, rapidjson::Document& allocator_provider`

Reply:
Renamed them. Not exactly to your suggestions, but I'm pretty sure there should no longer be any confusion about their purpose. [66ab7e5](https://github.com/apache/nifi-minifi-cpp/pull/1515/commits/66ab7e50bb0de7428b0cd5101e9189431c0119ac)